modules/ank/arrow-ank/src/main/kotlin/arrow/ank/main.kt

```diff
@@ -11,12 +10,12 @@ import java.nio.file.Paths
 fun main(vararg args: String) {
   when {
     args.size > 1 -> {
-      Try { ank(Paths.get(args[0]), Paths.get(args[1]), args.drop(2), monadDeferInterpreter(IO.monadDefer())).fix().unsafeRunSync() }
-        .fold({ ex ->
-          throw ex
-        }, { files ->
-          println("ΛNK Generated $files files")
-        })
+      ank(
+        Paths.get(args[0]),
+        Paths.get(args[1]),
+        args.drop(2),
+        monadDeferInterpreter(IO.monadDefer())
+      ).fix().unsafeRunSync()
```

Comment:
`unsafeAttemptSync()`? or `attempt().unsafeRunSync()`?. Can't remember which one we use for IO.

Reply:
@pakoito the gradle API requires throwing exceptions to fail a build.